pkg/teeth/sql/publisher_source.sql

```diff
@@ -0,0 +1,42 @@
+SELECT
+  pull_request_events.delivery_id,
+  delivery_events.pull_request_id,
+  html_url AS pull_request_html_url,
+  delivery_events.received,
+  logs_uri,
+  head_sha
+FROM
+  `%s` AS pull_request_events
+JOIN (
+  SELECT
+    delivery_id,
+    received,
+    logs_uri,
+    SAFE.INT64(pull_request.id) AS pull_request_id,
+    LAX_STRING(pull_request.url) AS pull_request_url,
+    LAX_STRING(events.payload.workflow_run.head_sha) AS head_sha,
+  FROM
+    `%s` leech_status
+  JOIN (
+	SELECT
+      *
+    FROM
+      `%s` events,
+      UNNEST(JSON_EXTRACT_ARRAY(events.payload.workflow_run.pull_requests)) AS pull_request
+    WHERE
+      received >= TIMESTAMP_ADD(CURRENT_TIMESTAMP(), INTERVAL -30 DAY)) AS events
+  USING
+    (delivery_id)) AS delivery_events
+ON
+  pull_request_events.id = delivery_events.pull_request_id
+WHERE
+  pull_request_events.id NOT IN (
+  SELECT
+    DISTINCT pull_request_id
+  FROM
+    `%s` invocation_comment_status)
```

Comment:
If we do keep %s, would it make more sense to use go-template so we can name these parameters?

Reply:
That's valid. I'll look back on any changes.

Reply:
Done.